CalendarApp/BookingRoom/Controllers/PostEventController.cs

```diff
@@ -19,8 +22,17 @@ public PostEventController(BookingService bookingService)
         [HttpPost]
         public HttpResponseMessage EventPost(CalendarEvent eventPost)
         {
-            BookingService.PostEvent(ToEventConverter.ToEvent(eventPost), eventPost.CalendarID);
+            List<Event> conflictedEvents = BookingService.ManagerEvent.CheckEvent(eventPost.CalendarID, ToEventConverter.ToEvent(eventPost), eventPost.ForcedEvent);
+            if (conflictedEvents.Count != 0)
+            {
+                var failRequest = new {status = "Unsuccesful", ConflictedList = conflictedEvents};
+                return Request.CreateResponse(HttpStatusCode.BadRequest, failRequest);
+            }
+            else
+            {
+                BookingService.PostEvent(ToEventConverter.ToEvent(eventPost), eventPost.CalendarID);
+            }                         
             return Request.CreateResponse(HttpStatusCode.Created, eventPost);
```

Comment:
You should return response in the same fashion that you are returning when sending conflicts